Mathlib/FieldTheory/Adjoin.lean

```diff
@@ -1285,16 +1325,32 @@ theorem fg_adjoin_finset (t : Finset E) : (adjoin F (↑t : Set E)).FG :=
 theorem fg_def {S : IntermediateField F E} : S.FG ↔ ∃ t : Set E, Set.Finite t ∧ adjoin F t = S :=
   Iff.symm Set.exists_finite_iff_finset
 
+theorem fg_adjoin_finite {t : Set E} (h : Set.Finite t) : (adjoin F t).FG :=
```

Comment:
Not `.._of_finite`?

Reply:
You can read it as "adjoin a finite set", but since there is (un-namespaced!) [fg_adjoin_of_finite](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/IntegralClosure/IsIntegral/Basic.html#fg_adjoin_of_finite), let me follow it:
```suggestion
theorem fg_adjoin_of_finite {t : Set E} (h : Set.Finite t) : (adjoin F t).FG :=
```